<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Config</class>
 <widget class="QWidget" name="Config">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>443</width>
    <height>190</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Color</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="1" column="1">
    <widget class="KComboBox" name="positioningCombo">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <item>
      <property name="text">
       <string>Scaled</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Centered</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Scaled and Cropped</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Tiled</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Center Tiled</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Maxpect</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="colorLabel">
     <property name="text">
      <string>Color:</string>
     </property>
     <property name="buddy">
      <cstring>colorButton</cstring>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="dayLabel">
     <property name="text">
      <string>Day:</string>
     </property>
     <property name="buddy">
      <cstring>dayCombo</cstring>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="nightLabel">
     <property name="text">
      <string>Night:</string>
     </property>
     <property name="buddy">
      <cstring>nightCombo</cstring>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="KPushButton" name="openButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Open...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KPushButton" name="getNewButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Get New Wallpapers...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="9" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="positioningLabel">
     <property name="minimumSize">
      <size>
       <width>125</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>Positioning:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>positioningCombo</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="KColorButton" name="colorButton"/>
   </item>
   <item row="4" column="1">
    <widget class="KComboBox" name="dayCombo">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="KComboBox" name="nightCombo">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KPushButton</class>
   <extends>QPushButton</extends>
   <header>kpushbutton.h</header>
  </customwidget>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>KColorButton</class>
   <extends>QPushButton</extends>
   <header>kcolorbutton.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>positioningCombo</tabstop>
  <tabstop>colorButton</tabstop>
  <tabstop>dayCombo</tabstop>
  <tabstop>nightCombo</tabstop>
  <tabstop>openButton</tabstop>
  <tabstop>getNewButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
